uktrade/data-hub-frontend
# Remove interaction form story

## Description of change

This PR removes an example usage of the interaction details form from storybook. This is due to it not being a reusable component but rather an implementation of the `TaskForm` component, for which there is already an entry in storybook. 

## Test instructions

Run storybook on this branch and check that there is no entry for `Company interactions`.

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
